applications/asset_tracker/src/main.c

```diff
@@ -1797,11 +1797,11 @@ static void ui_evt_handler(struct ui_evt evt)
 	if (IS_ENABLED(CONFIG_GPS_CONTROL_ON_LONG_PRESS) &&
 	   (evt.button == UI_BUTTON_1)) {
 		if (evt.type == UI_EVT_BUTTON_ACTIVE) {
-			k_delayed_work_submit_to_queue(&application_work_q,
+			k_work_reschedule_for_queue(&application_work_q,
 						       &long_press_button_work,
 						       K_SECONDS(5));
 		} else {
-			k_delayed_work_cancel(&long_press_button_work);
+			k_work_cancel_delayable(&long_press_button_work);
```

Comment:
What happens if this fails?

Reply:
the gps enabled state will be toggled if the work fails to be canceled